app/controllers/admin/token_packs_controller.rb

```diff
@@ -0,0 +1,46 @@
+module Admin
```

Comment:
Labak raksti class Admin::TokenPacksController < ....

Tev nakotne šie namespace skrasies un uz katru veidot bloku bus not cool